### PR TITLE
Backend/parts&orders

### PR DESCRIPTION
**Pull request by gkrzysztof**

### CODE REVIEW

File: car-workshop-backend/src/main/kotlin/bazydanych/model/Orders.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import bazydanych.model.user.UserId
+import kotlinx.serialization.Serializable
+
+typealias OrdersId = Id<Orders>
+
+@Serializable
+data class Orders(
+    val id: OrdersId,
+    val ownerId: UserId,
+    val Id_Parts: Int,
```
Comment (flkrystian) [inline car-workshop-backend/src/main/kotlin/bazydanych/model/Orders.kt:12]: partId

File: car-workshop-backend/src/main/kotlin/bazydanych/model/Orders.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    val Status: String,
```
Comment (flkrystian) [inline car-workshop-backend/src/main/kotlin/bazydanych/model/Orders.kt:13]: status

File: car-workshop-backend/src/main/kotlin/bazydanych/service/form/PartsCreateForm.kt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package bazydanych.service.form
+
+import bazydanych.util.BigDecimalSerializer
+import kotlinx.serialization.Serializable
+import java.math.BigDecimal
+
+@Serializable
+data class PartsCreateForm(
+    val id_Vehicle: Int,
+    val product_name: String,
```
Comment (flkrystian) [inline car-workshop-backend/src/main/kotlin/bazydanych/service/form/PartsCreateForm.kt:10]: camelCase

**Review by flkrystian — CHANGES_REQUESTED** [review]

**Review by flkrystian — APPROVED** [review]

**Comment by gkrzysztof** [comment]
mamy to

**Pull request merged**